pages/Documents/Security&Authentication/ConsumerAuthentication/implementation-guide.md

```diff
@@ -1,17 +1,17 @@
 ---
-pagename: Detailed API
+pagename: Implementation Guide
 redirect_from:
```

Comment:
Please add a redirect from the old file name (consumer-authentication-detailed-api.html).

Reply:
@lpbarber, if you could make this change we can roll out the update.